doc/en/skipping.rst

```diff
@@ -293,6 +293,18 @@ imperatively, in test or setup code::
             # or
             pytest.skip("unsupported configuration")
 
+Note that calling ``pytest.skip`` at the module level 
+is not allowed since pytest 3.0. To skip
+all tests in a module given some runtime condition, you can
+set a ``pytestmark`` variable:
+
+.. code-block:: python
+
+    if SOME_CONDITION:
```

Comment:
I agree, but that was just a recipe on how to replace `pytest.skip` with a mark at module level. 

Perhaps this then?

```python
try:
    my_module.check_some_condition()
except ConditionUnsatisfied:
    pytestmark = pytest.mark.skip('condition not satisfied' 
```